lib/configuration.rb

```diff
@@ -16,7 +16,7 @@ module Configuration
   def which(cmd, extra_paths = nil)
     exts = ENV['PATHEXT'] ? ENV['PATHEXT'].split(';') : ['']
     path_array = ENV['PATH'].split(File::PATH_SEPARATOR)
-    path_array = path_array.concat(extra_paths) unless extra_paths.nil?
+    path_array.concat(extra_paths) unless extra_paths.nil?
```

Comment:
So apparently `Array.concat` modifies the variable in place, so we didn't need the redundant assignment.  https://ruby-doc.org/core-2.7.0/Array.html#method-i-concat